.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,53 @@
+---
+name: Bug report
+description: Create a report to help us improve.
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for opening a bug report towards Prometheus.
+
+        Please do *NOT* ask support questions in Github issues.
+
+        If your issue is not a feature request or bug report use our [community support](https://prometheus.io/community/).
+
+        There is also [commercial support](https://prometheus.io/support-training/) available.
+  - type: textarea
+    attributes:
```

Comment:
I would add that : 

```yaml
    validations:
      required: true
```

Reply:
and same for `What did you expect to see?` and `What did you see instead? Under which circumstances?` sections